hooks/useLockout.ts

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Gridiron Survivor.
+// Licensed under the MIT License.
+
+import { useEffect, useState } from 'react';
+
+/**
+ * Function to constantly check the current time every hour and set the lockout accordingly.
+ * @returns - The value of lockedOut.
+ */
+const useLockout = (): boolean => {
+  const [lockedOut, setLockedOut] = useState<boolean>(false);
+  useEffect(() => {
+    /**
+     * Checks if the user is locked out from making a pick.
+     */
+    const checkLockout = (): void => {
+      const currentDateAndTime = new Date();
+      const day = currentDateAndTime.getUTCDay();
+      const hours = currentDateAndTime.getUTCHours();
+      if (
+        (day === 5 && hours >= 0) || // Thursday at 9pm EST
```

Comment:
Do we care about time zones? I'm still ramping up and this may be a well-known answer but one I think is worth me asking.

Reply:
these numbers are UTC time which is 4 hours ahead of me. I left the comments for easy visual understanding. It handles when the lockdown period is.

Reply:
> Do we care about time zones? I'm still ramping up and this may be a well-known answer but one I think is worth me asking.

@braydoncoyer I used UTC so we don't have to worry about time zones. I only left the comments in there for reference, like Alex said.